pkg/accounting/tally/tally_test.go

```diff
@@ -67,6 +72,8 @@ func sendGeneratedAgreements(ctx context.Context, t *testing.T, planet *testplan
 	for i, action := range actions {
 		pba, err := testbwagreement.GeneratePayerBandwidthAllocation(action, satID, upID, time.Hour)
 		require.NoError(t, err)
+		err = testbwagreement.SavePayerBandwidthAllocation(db.CertDB(), pba, upID.Leaf.PublicKey.(*ecdsa.PublicKey))
```

Comment:
nit: Based on the function name "sendGeneratedAgreements", I would move this call to outside of sendGeneratedAgreements(), to increase that clarity that saving the uplink cert on the server is not related to the storage node sending the bandwidth agreement to the satellite.

Reply:
I looked at the changes to fix this. this is what I am thinking. If I have to pull out "SavePayerBandwidthAllocation() out of the function sendGeneratedAgreements(), then I have to change the sendGeneratedAgreements() signature to return a slice of *psdb.Agreement and also move SendAgreementsToSatellite() out.  It is bit of changes. Is this change you referring to? am I getting it right?

Reply:
I think you're getting tripped up by the misleading naming of 'SavePayerBandwidthAllocation()'.  All that function does is save the uplink cert.  We can do that once, rather than for each bandwidth agreement.